conftest.py

```diff
@@ -237,3 +239,54 @@ def remote_file(request):
 
     session.delete(new_connection)
     session.flush()
+
+
+@pytest.fixture
+def database_table_fixture(request):
+    params = request.param
+    table = params.get("table", NewTable())
+    filepath = params.get("filepath", "")
+
+    database_name = params["database"]
+    database_name_to_conn_id = {Database.SQLITE: "sqlite_default"}
+    conn_id = database_name_to_conn_id[database_name]
+    database = get_database_from_conn_id(conn_id)
+
+    database.drop_table(table)
+    if filepath:
+        database.load_file_to_table(filepath, table)
+    yield database, table
+
+    database.drop_table(table)
+
+
+@pytest.fixture
+def cloud_object_fixture(request):
```

Comment:
Same here, would be good to have a comment on what this fixture is for and when to use it

Reply:
I think there was another fixture dealing with the 'remote_file'. Can we have one fixture to cater to all remote file needs?